mithril-common/src/entities/signer.rs

```diff
@@ -1,25 +1,60 @@
-use crate::entities::{PartyId, Stake};
+use crate::{
+    crypto_helper::KESPeriod,
+    entities::{HexEncodedKey, PartyId, Stake},
+};
+
 use serde::{Deserialize, Serialize};
 use sha2::{Digest, Sha256};
 
 /// Signer represents a signing participant in the network
 #[derive(Clone, Debug, PartialEq, Eq, Default, Serialize, Deserialize)]
 pub struct Signer {
     /// The unique identifier of the signer
+    // TODO: Should be removed once the signer certification is fully deployed
     #[serde(rename = "party_id")]
     pub party_id: PartyId,
 
     /// The public key used to authenticate signer signature
     #[serde(rename = "verification_key")]
-    pub verification_key: String,
+    pub verification_key: HexEncodedKey,
+
+    /// The encoded signer 'Mithril verification key' signature (signed by the Cardano node KES secret key)
+    // TODO: Option should be removed once the signer certification is fully deployed
+    #[serde(
+        rename = "verification_key_signature",
```

Comment:
why are we renaming if it is the same name? I see this is done in all fields, so it might be intentional.

Reply:
You're right, this not necessary to use this annotation, I removed it for all the fields in this `signer.rs` module :heavy_check_mark:

We'll remove the other ones that are not necessary in the other entities sub modules in a different PR